packages/zowe-explorer/src/Profiles.ts

```diff
@@ -1330,7 +1329,6 @@ export class Profiles extends ProfilesCache {
                     ZoweLogger.info(successMsg);
                     Gui.showMessage(successMsg);
                     await this.tokenAuthClearSecureArray(serviceProfile.name);
-                    await vscode.commands.executeCommand("zowe.extRefresh");
```

Comment:
Can you please add this logic after line 1331 to fire the `onProfilesUpdate` event?

```ts
ZoweExplorerApiRegister.getInstance().onProfilesUpdateEmitter.fire(EventTypes.UPDATE);
```

Thanks!